### DISCUSSION TITLE
Test Results in Panel

### DISCUSSION THREAD

**Discussion by ReGrantDev** [orta/vscode-jest#1057] [General]
Every time I run tests with the plugin, the bottom panel opens up and focuses the TEST RESULTS which I find very distracting, is there a way to make it not automatically do this (I'm fine opening it manually as I need it).

**Comment by connectdotz** [comment]
Yes, there is, please see this [comment](https://github.com/jest-community/vscode-jest/issues/1051#issuecomment-1676143068).
